tests/_utils-tests/model.js

```diff
@@ -12,6 +12,7 @@ import Element from '/ckeditor5/engine/model/element.js';
 import Text from '/ckeditor5/engine/model/text.js';
 import Range from '/ckeditor5/engine/model/range.js';
 import Position from '/ckeditor5/engine/model/position.js';
+import CKEditorError from '/ckeditor5/utils/ckeditorerror.js';
```

Comment:
>  it was meant to be minified in a special way.

I think this was the main reason. Using `CKEditorError` inside tests will probably do no harm as long as these errors will not be confused with errors from source code (and not included in minification). I wanted to make this fix to be consistent with the rest of the file.